src/app/components/add-todo/add-todo.component.ts

```diff
@@ -8,19 +8,27 @@ import { TodosStoreService } from 'src/app/services/todos-store.service';
   templateUrl: './add-todo.component.html',
   styleUrls: ['./add-todo.component.scss'],
 })
-export class AddTodoComponent {
+export class AddTodoComponent implements OnInit {
   @ViewChild('formDir') private formDir: NgForm;
   @ViewChild('inputEl') private inputEl: ElementRef;
 
+  minDate: Date;
+
   addForm = this.fb.group({
     title: ['', [Validators.maxLength(100), Validators.required]],
+    deadline: [this.minDate, [Validators.required]],
   });
 
   constructor(private todosStore: TodosStoreService, private fb: FormBuilder) {}
 
+  ngOnInit(): void {
+    this.minDate = new Date();
+  }
+
   onSubmit(): void {
     const title = this.addForm.get('title').value;
-    this.todosStore.add(title);
+    const deadline = this.addForm.get('deadline').value;
+    this.todosStore.add({ title, deadline });
```

Comment:
it looks like the `this.addForm.value` matches the argument of `this.todosStore.add` method. So, intermediate `title` and `deadline` variables can be omitted

Reply:
So, doing the following is preferred here, correct?

`this.todosStore.add(this.addForm.value);`

Reply:
yes, it can be in that way